app/controllers/vendor_api/notes_controller.rb

```diff
@@ -0,0 +1,36 @@
+module VendorAPI
+  class NotesController < VendorAPIController
+    before_action :validate_metadata!
+
+    include ApplicationDataConcerns
+
+    def create
+      @new_note_form = AllowedCrossNamespaceUsage::ProviderInterfaceNewNoteForm.new(note_params)
+
+      if @new_note_form.save
+        application_choice = application_choices_visible_to_provider.find(params[:application_id])
+
+        render json: %({"data":#{ApplicationPresenter.new(version_number, application_choice).serialized_json}})
```

Comment:
```suggestion
        render_application
```
https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/controllers/concerns/vendor_api/application_data_concerns.rb#L13

`application.reload` might be needed before rendering the application or we can consider updating [the](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/controllers/concerns/vendor_api/application_data_concerns.rb#L9) so it always retrieves the application.